tests/integrations/aws_lambda/test_aws.py

```diff
@@ -487,6 +490,15 @@ def test_handler(event, context):
         ),
         (b"[]", False, 1),
     ],
+    ids=[
```

Comment:
don't understand what this is for

Reply:
those are just some identifiers for the pytest output. so it is easier to see which ones are failing.